### ISSUE TITLE
Math in marginnote (and change of MathJax id)

### ISSUE THREAD

**Issue by ekstroem** [clayh53/tufte-jekyll#36]
Hi.

This may just be a liquid problem (or rather a user problem of not understanding the liquid tags) but I'd like to add a wee bit of inline math to a margin note with the following code

    {% marginnote 'ID9' 'Yada yada yada. A {% m %} X^2 {% em %} test for this table yada yada yada' %}

However, that results in an error

>  Liquid Exception: Unmatched double quote: 

and I'm guessing that I cannot nest liquid tags inside one another. 

One option is to change the math start and end tags to $ (for inline) and $$ (for block) as is possible with other configurations of MathJax. Would doing that break a lot of things (and if no: can anyone point me in the right direction for this)?

**Comment by clayh53** [comment]
let me look into it. The issue may be a 'tags within tags' thing. But if we can just use the default '$' and '$$' tags that are baked into Mathjax, that may be the way to go.

**Comment by xHN35RQ** [comment]
So I hadn't looked into this before, but now that I do, it turns out Jekyll's default Markdown renderer has built-in MathJax support http://kramdown.gettalong.org/math_engine/mathjax.html

So @ekstroem either of these snippets work great!

```
{% marginnote 'ID9' 'Yada yada yada. A $X^2$ test for this table yada yada yada' %}
```
```
{% marginnote 'ID9' 'Yada yada yada. A \\(X^2\\) test for this table yada yada yada' %}
```

@clayh53 There is no need for the `mathjaxtag.rb` plugin! Using `$` and `$$` for equations is easy to setup by how you load mathjax into the page. See http://docs.mathjax.org/en/latest/start.html#tex-and-latex-input

**Comment by clayh53** [comment]
Oh, snap! I *do* remember reading some documentation about Kramdown having this ability. That makes it easy!

**Comment by ekstroem** [comment]
Great! I can confirm that 

    {% marginnote 'ID9' 'Yada yada yada. A $X^2$ test for this table yada yada yada' %}

works. I previously had some problems with $ signs in the main text (in the sense that they were rendered as dollar signs and not starting/ending math blocks) - I'll try to see if I can reproduce this. In any case: this works!

Cheers

**Comment by xHN35RQ** [comment]
Using `$` for inline equations is disabled by default in mathjax See https://github.com/clayh53/tufte-jekyll/issues/38 for how to enable it.

**Comment by ekstroem** [comment]
Hmm. It works out of the box here at my end (what I really mean is that I cannot remember what I've tinkered with previously but I cannot see a change like fix #38 anywhere. In any case there's a good solution, and I'm happy :)

**Comment by xHN35RQ** [comment]
Ok cool.
